CountlyTests/CountlyBaseTestCase.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  CountlyBaseTestCase.swift
+//  CountlyTests
+//
+//  Created by Muhammad Junaid Akram on 27/12/2023.
+//  Copyright © 2023 Alin Radut. All rights reserved.
+//
+
+import XCTest
+@testable import Countly
+
+class CountlyBaseTestCase: XCTestCase {
+    var countly: Countly!
+    var deviceID: String = ""
+    let appKey: String = "58594c9a3f461ebc000761a68c2146659ef75ea0"
+    var host: String = "https://master.count.ly/"
+    
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+        let config: CountlyConfig = CountlyConfig()
+        config.appKey = appKey
+        config.host = host
+        countly = Countly();
+        countly.start(with: config)
+        //         let queuedRequests = CountlyPersistency.sharedInstance().value(forKey: "queuedRequests") as? NSMutableArray
```

Comment:
this was to get the queue?

Reply:
yes, was added it just for testing, will remove it from here.